src/utils/web3/ledgerConnector.ts

```diff
@@ -122,7 +122,7 @@ export class IFrameEthereumConnector extends Connector<IFrameEthereumProvider, I
 			account: toAddress(account),
 			chain,
 			transport: custom(provider)
-		});
+		}) as WalletClient;
```

Comment:
`WalletClient` from viem and wagmi are slightly different, I'm ashamed to do this but for now it's not too bad.

We can try and address this later